nixos/tests/nixos-test-driver/coverage-driver.nix

```diff
@@ -0,0 +1,39 @@
+pkgs: driver:
+let
+  coverage-rc = pkgs.writeText "coverage.rc" ''
+    [run]
+    include =
+        **/site-packages/test_driver/*
+  '';
+
+  coverage = "${pkgs.python3Packages.coverage}/bin/coverage";
+
+  pythonButItsCoverage = pkgs.writeScript "python-coverage" ''
+    #!${pkgs.runtimeShell}
+    set -euxo pipefail
+    COVERAGE_RCFILE=${coverage-rc} ${coverage} run -- "$@"
+    RETURN_CODE=$?
+
+    if whoami | grep "nixbld"; then
```

Comment:
This is actually incorrect… In auto-uid-allocation, nixbld does not appear in the username.